### ISSUE TITLE
How to save a certain selected object/objects

### ISSUE THREAD

**Issue by mhasil** [austynmahoney/mobile-export-scripts-illustrator#13]
This script is excellent. Especially the folder creation and automatic naming is very handy. It would be very helpful if there were a way to save a selected object on the artboard instead of the entire artboard. Also, the ability to change the file name would be very useful. Finally, there should be a way to save an icon that is made up of multiple objects; this is very important for us designers.
To summarize, I am looking for something similar to the "export selection" option in the file menu.

**Comment by austynmahoney** [comment]
Thank you @mhasil.

The file name is set to the artboard name when you export. Is there a use case that doesn't solve?

Selection does sound like a good idea, but I have moved onto SVG (`VectorDrawables` on Android) files in my apps now. Pull requests for these features are always welcome.

**Comment by austynmahoney** [comment]
Closing this due to inactivity.
